demo/api/package.json

```diff
@@ -17,8 +17,8 @@
         "mikro-orm": "mikro-orm",
         "mikro-orm:drop": "mikro-orm schema:drop -r",
         "mikro-orm:migration:generate": "mikro-orm migration:create",
-        "db:migrate:prod": "yarn console:prod migrate --",
-        "db:migrate": "yarn console migrate --",
+        "db:migrate:prod": "yarn api-generator && yarn console:prod migrate --",
+        "db:migrate": "yarn api-generator && yarn console migrate --",
```

Comment:
@nsams Do you think it's necessary to execute the `api-generator` before every `console` command?

Reply:
1. it's not required at all for prod
2. in theory it's required before every console, but it will make it very slow. in practice it should be ok like that